src/components/Sidebar/index.js

```diff
@@ -8,12 +8,13 @@ import { EXPAND_MENU } from './consts';
 const Sidebar = ({
```

Comment:
Convert to TS (if you get stuck I can help!)

Reply:
Have had a go at TS definitions for this and RightSidebar now!